src/singletons/Settings.hpp

```diff
@@ -677,11 +677,10 @@ class Settings
     bool isMutedChannel(const QString &channelName);
     bool toggleMutedChannel(const QString &channelName);
     std::optional<QString> matchNickname(const QString &username);
-
-private:
     void mute(const QString &channelName);
     void unmute(const QString &channelName);
```

Comment:
If you make these public, please ensure that the channel name only occurs once in the `mutedChannels` (e.g. when repeatedly calling `mute("foo")`).